### ISSUE TITLE
Feature request: ability to filter billing page by project/environment

### ISSUE THREAD

**Issue by abigailbramble** [PostHog/posthog#26171]
## Is your feature request related to a problem?

Would be great if users could gain insight into how their billing is broken down by project/environment.

## Additional context
https://posthoghelp.zendesk.com/agent/tickets/20353

**Comment by davecranwell** [comment]
Yes this would be particularly helpful when diagnosing the origin of certain costs, esp. if they seem run-away. Knowing which of your sites contributed to the charges might make for easier choices about where to focus development
